### PR TITLE
EZP-28890 Added Crowdin in-context translation

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZP-28890
| Versions      | 2.3

Added Crowdin in-context translation

### Crowdin in-context translation

AdminUI interface is now translatable using [Crowdin's in-context tool](../community_resources/translations#how-to-translate-the-interface-using-crowdin). When navigating through the AdminUI interface you will now be able to translate it. A glossary has been established to aid in unified usage of terminology throughout. Contributions are welcome - https://crowdin.com/project/ezplatform

### CODE REVIEW

**Comment by MagdalenaZuba** [comment]
eZ Platform Crowdin project is only accessible to eZ employees for now as it is a test phase. It is yet to be opened to the public (possibly in v2.3).

File: docs/community_resources/translations.md [hunk 1]
```diff
@@ -5,6 +5,54 @@ If you'd like to see eZ Platform in your language, you can contribute to the tra
 [`ezplatform-i18n`](https://github.com/ezsystems/ezplatform-i18n) contains the XLIFF files providing translations.
 You can use an XLIFF editor of your choice to contribute strings in your language.
 
+## How to translate the interface using Crowdin
+
+If you wish to contribute to an existing translation of AdminUI or start a new one, the best way is to start with in-context translation (but you can also [translate directly on the Crowdin website](#translating-outside-in-context)).
```
Comment (DominikaK) [inline docs/community_resources/translations.md]: `AdminUI` -> `Back Office`

File: docs/community_resources/translations.md [hunk 2]
```diff
@@ -5,6 +5,54 @@ If you'd like to see eZ Platform in your language, you can contribute to the tra
...
+**URL**: `javascript:function hasInContextCookie(){ return document.cookie.match(/^(.*;)?\s*ez_in_context_translation\s*=\s*[^;]+(.*)?$/);}(function () { document.cookie = hasInContextCookie() ? document.cookie = &#39;ez_in_context_translation=;expires=Mon, 05 Jul 2000 00:00:00 GMT;path=/;&#39;: document.cookie=&#39;ez_in_context_translation=1;path=/;&#39;; location.reload()})()`
+
+Enter AdminUI. While on the page clicking this bookmark will turn on and off in-context translation for eZ Platform interface.
+
+### Using the debugging console
+
+Another way is to open the development console and run this command:
+
+``` javascript
+function hasInContextCookie(){ return document.cookie.match(/^(.*;)?\s*ez_in_context_translation\s*=\s*[^;]+(.*)?$/);}(function () { document.cookie = hasInContextCookie() ? document.cookie = 'ez_in_context_translation=;expires=Mon, 05 Jul 2000 00:00:00 GMT;path=/;': document.cookie='ez_in_context_translation=1;path=/;'; location.reload()})()
```
Comment (DominikaK) [inline docs/community_resources/translations.md]: Maybe use the shorter command from here: https://discuss.ezplatform.com/t/how-to-help-to-translate-ez-platform-v2/425

File: docs/community_resources/translations.md [hunk 3]
```diff
@@ -5,6 +5,54 @@ If you'd like to see eZ Platform in your language, you can contribute to the tra
...
+
+Right-click your browser's bookmark bar, and create a new bookmark as shown in the screenshot below:
+
+![Bookmark for toggling in-context translation](img/toggle_incontext.png "Bookmark for toggling in-context translation")
+
+**Name**: Toggle in-context
+
+**URL**: `javascript:function hasInContextCookie(){ return document.cookie.match(/^(.*;)?\s*ez_in_context_translation\s*=\s*[^;]+(.*)?$/);}(function () { document.cookie = hasInContextCookie() ? document.cookie = &#39;ez_in_context_translation=;expires=Mon, 05 Jul 2000 00:00:00 GMT;path=/;&#39;: document.cookie=&#39;ez_in_context_translation=1;path=/;&#39;; location.reload()})()`
+
+Enter AdminUI. While on the page clicking this bookmark will turn on and off in-context translation for eZ Platform interface.
```
Comment (DominikaK) [inline docs/community_resources/translations.md]: Back Office

File: docs/community_resources/translations.md [hunk 4]
```diff
@@ -5,6 +5,54 @@ If you'd like to see eZ Platform in your language, you can contribute to the tra
 [`ezplatform-i18n`](https://github.com/ezsystems/ezplatform-i18n) contains the XLIFF files providing translations.
 You can use an XLIFF editor of your choice to contribute strings in your language.
 
+## How to translate the interface using Crowdin
+
+If you wish to contribute to an existing translation of Back Office or start a new one, the best way is to start with in-context translation (but you can also [translate directly on the Crowdin website](#translating-outside-in-context)).
```
Comment (bdunogier) [inline docs/community_resources/translations.md]: I'd split that into two sentences. The interface is important as well, even if we first mention in-context.
We can also reference the translation platform @SylvainGuittard has configured.

File: docs/community_resources/translations.md [hunk 5]
```diff
@@ -5,6 +5,54 @@ If you'd like to see eZ Platform in your language, you can contribute to the tra
...
+
+## Preparing to use in-context
+
+To start translating, you need an option to turn in-context translation on and off. To do this, set a browser cookie. There are several ways to do this, we will highlight a couple here.
```
Comment (bdunogier) [inline docs/community_resources/translations.md]: To make it more accurate: "To enable the in-context translation UI, a pseudo-language locale needs to be used in HTTP requests. To make this easier, a cookie `ez_in_context_translation=1` can be set. There are several ways to do this. We will highlight a couple down below."

**Pull request merged**